### PR TITLE
Implement dequantizeLinear

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin @fdwr @lisa0314 PTAL, thanks.

### CODE REVIEW

File: test/dequantize_linear_test.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            2,
+            4,
+          ],
+        },
+        { // zeroPoint
+          shape: [3, 1],
+          value: [
+            0,
+            0,
+            0,
```
Comment (fdwr) [inline test/dequantize_linear_test.js:99]: Results would be a little more interesting to include some nonzero values. e.g.

```
          value: [
            1,
            2,
            3,
          ],
```

File: test/dequantize_linear_test.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    const scaleTensor = new Tensor(scale.shape, scale.value);
+    const zeroPointTensor = new Tensor(zeroPoint.shape, zeroPoint.value);
+    const outputTensor = dequantizeLinear(inputTensor, scaleTensor, zeroPointTensor);
+    utils.checkShape(outputTensor, expected.shape);
+    utils.checkValue(outputTensor, expected.value);
+  }
+
+
+  it('dequantizeLinear 1D broadcasting scale and zeroPoint', function() {
+    testDequantizeLinear(
```
Comment (fdwr) [inline test/dequantize_linear_test.js:19]: A 0D case would complete the set.

**Review by fdwr — COMMENTED** [review]
Minor comments, else LGTM.

**Comment by BruceDai** [comment]
@fdwr I've updated tests in new commit, please take another look, thanks.

**Review by fdwr — APPROVED** [review]
👍

**Review by huningxin — APPROVED** [review]

**Pull request merged**